clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/userdata/UserDataProvider.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.spinnaker.clouddriver.aws.deploy.userdata;
```

Comment:
ah - didn't notice that. was an intellij convert to java guess it lost the header (I'll add it back)